internal/router/router.go

```diff
@@ -3,14 +3,14 @@ package router
 import (
 	"github.com/fabric8-services/fabric8-jenkins-proxy/internal/api"
 	"github.com/julienschmidt/httprouter"
-	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 // CreateAPIRouter is creating a router for the REST API of the Proxy.
 func CreateAPIRouter(api api.ProxyAPI) *httprouter.Router {
 	// Create router for API
 	proxyRouter := httprouter.New()
 	proxyRouter.GET("/api/info/:namespace", api.Info)
-	proxyRouter.Handler("GET", "/metrics", prometheus.Handler())
+	proxyRouter.Handler("GET", "/metrics", promhttp.Handler())
```

Comment:
Don't there is need to change package to work go build?
Here somewhere - https://github.com/fabric8-services/fabric8-jenkins-proxy/blob/master/Gopkg.lock#L190

Reply:
I think that since we added the base package (prometheus client) the promhttp is automatically added as well (because it is a subpackage) so wedo not need to update anything as @chmouel noted.Also it compiles in CI env so it seems everything is correct. @nurali-techie are you ok on merging it?